mappings/roman/roman_0005.pmap

```diff
@@ -0,0 +1,13 @@
+header = {
+    'derived_from' : 'roman_0004.pmap',
+    'description' : 'Initial delivery',
```

Comment:
The description field would be a nice place to document why this context exists (it contains a bad file).  I'd also recommend something similar in the file's meta.description, so that developers who come along later know why the file exists in the test cache.